wal.go

```diff
@@ -15,6 +16,10 @@ const (
 	initialSegmentFileID = 1
 )
 
+var (
+	ErrTooLarge = errors.New("the data size can't larger than sigment size")
```

Comment:
Maybe a ambiguous name, `ErrDataTooLarge` or `ErrValueTooLarge`

Reply:
go it